lib/ansible/modules/net_tools/nios/nios_dns_view.py

```diff
@@ -26,8 +26,9 @@
 options:
   name:
     description:
-      - Specifies the name of the DNS view to add and/or remove from the
-        system configuration based on the setting of the C(state) argument.
+      - Specifies the name of the DNS view to add and/or remove and/or update
+        from the system configuration based on the setting of the C(state)
+        argument.
```

Comment:
Please add documentation for valid value type for this option.
It can accept either value of type string or if user want to update existing name can accept
a value of type dict in format {'new_name': '<new-name>', 'old_name': '<old-name>'}